src/DankCord/DankCord.py

```diff
@@ -142,62 +158,70 @@ def run_sub_command(self, /, name : str, sub_name : str, retry = False, attempts
         type_ = item["type"]
         break
     data = {
-      "type": 2,
-      "application_id": "270904126974590976",
-      "guild_id": self.guild_id,
-      "channel_id": self.channel_id,
-      "session_id": self.session_id,
-      "data": {
-        "version": command_info["version"],
-        "id": command_info["id"],
-        "name": name,
-        "type": command_info["type"],
-        "options": self._OptionsBuilder(sub_name, type_, **kwargs),
-        "application_command": {
-          "id": command_info["id"],
-          "application_id": "270904126974590976",
-          "version": command_info["version"],
-          "default_permission": command_info["default_permission"],
-          "default_member_permissions": command_info["default_member_permissions"],
-          "type": command_info["type"],
-          "name": name,
-          "description": command_info["description"],
-          "dm_permission": command_info["dm_permission"],
-          "options": command_info["options"]
+        "type": 2,
+        "application_id": "270904126974590976",
+        "guild_id": self.guild_id,
+        "channel_id": self.channel_id,
+        "session_id": self.session_id,
+        "data": {
+            "version": command_info["version"],
+            "id": command_info["id"],
+            "name": name,
+            "type": command_info["type"],
+            "options": self._OptionsBuilder(sub_name, type_, **kwargs),
+            "application_command": {
+                "id": command_info["id"],
+                "application_id": "270904126974590976",
+                "version": command_info["version"],
+                "default_permission": command_info["default_permission"],
+                "default_member_permissions": command_info["default_member_permissions"],
+                "type": command_info["type"],
+                "name": name,
+                "description": command_info["description"],
+                "dm_permission": command_info["dm_permission"],
+                "options": command_info["options"]
+            },
+            "attachments": []
         },
-        "attachments": []
-      },
-      "nonce": nonce
+        "nonce": nonce
     }
-    print(data)
+    message_data = None
+    message_object = False
+    raw_name = name + " " + sub_name
     for i in range(attempts):
-      response = Response(requests.post("https://discord.com/api/v9/interactions", json.dumps(data), http_headers=[("Authorization", self.token), ("Content-Type", "application/json")]))    
-      status = self._confirm_command_ran(nonce, timeout)
+      request = requests.post("https://discord.com/api/v9/interactions", json.dumps(
```

Comment:
Please format this properly, each param per line or singleline it

Reply:
Done